src/shared/default-routes.js

```diff
@@ -40,16 +41,16 @@ const DefaultRoutes = ({
       }
       <Route
         path='/'
-        component={FrontPage || Page}  />
+        component={fetchData(FrontPage || Page || Error)}  />
       <Route
         path=':page(/:subpage)'
-        component={Page} />
+        component={fetchData(Page || Error)} />
```

Comment:
There was a line `const components.Error || Missing View` above, which I think was for supplying your own Error component from the User Project.